.github/test-spec.yml

```diff
@@ -84,7 +84,20 @@
   - "**/*"
 
 "CI-matter-test":
-  - "**/*"
+  - "include/dfu/**/*"
+  - "include/mgmt/mcumgr/**/*"
+  - "soc/arm/nordic_nrf/**/*"
+  - "subsys/dfu/**/*"
+  - "subsys/settings/**/*"
+  - "subsys/net/**/*"
+  - "subsys/mgmt/mcumgr/**/*"
+  - "drivers/net/**/*"
+  - "drivers/wifi/**/*"
+  - "samples/hci_rpmsg/**/*"
+  - "samples/subsys/mgmt/mcumgr/smp_svr/**/*"
+  - any:
+    - "subsys/bluetooth/**/*"
```

Comment:
probably only `bluetooth/host` and `bluetooth/controller` are needed? At least `bluetooth/audio` shouldn't trigger our tests.